copper-rx2/src/androidTest/java/app/cash/copper/rx2/RxContentResolverTest.java

```diff
@@ -56,7 +54,8 @@ public RxContentResolverTest() {
   }
 
   public void testCreateQueryObservesInsert() {
-    db.createQuery(TABLE, null, null, null, null, false).subscribe(o);
+    RxContentResolver.observeQuery(contentResolver, TABLE, null, null, null, null, false, scheduler)
```

Comment:
These kinda suck because they're still in Java. Even with `@JvmOverloads` I need to pass the last argument, the scheduler, so it would look the same.

In Kotlin it's 👍